spec/factories/jobs.rb

```diff
@@ -1,8 +1,9 @@
 FactoryGirl.define do
   factory :job do
-    state "MyString"
-    title "MyString"
-    description "MyText"
-    organization_id 1
+    state 'draft'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.